commentTone/README.md

```diff
@@ -1,14 +1,143 @@
 # Comment tone
```

Comment:
Please take a look at the example README in the issue. You can put the other content in yopur `doc` folder if you don't want to delete the whole content.